adam/experiment/run_pipeline.py

```diff
@@ -0,0 +1,646 @@
+"""
+A script intended to run the entire M3 pipeline.
```

Comment:
Is this supposed to be Phase 3 or milestone 3?

Reply:
Phase 3. Whoops.